src/config/initChart.js

```diff
@@ -44,7 +44,6 @@ export default ({ pipSize = 0, type = 'area', noData = false, theme = 'light', s
             startOnTick: false,
             endOnTick: false,
             crosshair: false,
-            ordinal: false,
```

Comment:
with ordinal false, our strategy of using a separate series for future data does not work, as it will render a super huge white space

on the other hand, ordinal true will remove gap, meaning we cannot visualize closing time using a gap, need to dig more